olp-cpp-sdk-core/src/cache/ProtectedData.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include "olp/core/cache/DefaultCache.h"
+
+#include <set>
+#include <string>
+
+namespace olp {
+namespace cache {
+
+class ProtectedData {
```

Comment:
What you are actually protecting is a key list. So I would call it `ProtectedKeyList`